test/MUXControlsTestApp/ThemeResourcesTests.cs

```diff
@@ -423,6 +423,7 @@ private bool CompareObjects(object value, object mergedValue, int indentation)
 
                     PropertyInfo[] propertyInfoList = null;
 
+
                     if (valueObject != null && mergedValueObject != null)
```

Comment:
Run "git checkout origin/master ThemeResourcesTests.cs" to revert your changes in this branch